### PR TITLE
Issue 16 🐛 fix timeouts for Freenom World DNS in DE by using second server instead of first server

### PR DESCRIPTION
**Pull request by thomasmerz**
This PR solves issue #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The DNS services currently used for sending queries are listed below:
 | [AdGuard][AdGuard] (`94.140.14.140`)           | [CleanBrowsing][CleanBrowsing] (`185.228.168.9`) | [AdGuard][AdGuard] (`94.140.14.14`)         |
 | [Cloudflare][Cloudflare] (`1.1.1.1`)           | [Cloudflare][Cloudflare] (`1.1.1.2`)             | [AhaDNS][AhaDNS] (`5.2.75.75`)              |
 | [dns0.eu][dns0.eu] (`193.110.81.254`)          | [Comodo][Comodo] (`8.26.56.26`)                  | [CONTROL D][CONTROL D] (`76.76.2.2`)        |
-| [Freenom World][Freenom World] (`80.80.80.80`) | [CONTROL D][CONTROL D] (`76.76.2.1`)             | [dnsforge.de][dnsforge.de] (`176.9.93.198`) |
+| [Freenom World][Freenom World] (`80.80.81.81`) | [CONTROL D][CONTROL D] (`76.76.2.1`)             | [dnsforge.de][dnsforge.de] (`49.12.223.2`) |
```
Comment (PeterDaveHello) [inline README.md:15]: dnsforge.de is not related here

File: chkdm [hunk 2]
```diff
@@ -64,32 +64,32 @@ declare -A nofilterDNS secureDNS adblockDNS
...
 secureDNS[OpenDNS]="208.67.222.222"
 secureDNS[Quad101]="101.101.101.101"
 secureDNS[Quad9]="9.9.9.9"
+secureDNS[SafeDNS]="195.46.39.39"
+secureDNS[UltraDNS]="156.154.70.2"
 secureDNS[Yandex]="77.88.8.2"
 
 adblockDNS[AdGuard]="94.140.14.14"
 adblockDNS[AhaDNS]="5.2.75.75"
+adblockDNS[clean.dnsforge.de]="49.12.223.2"
```
Comment (PeterDaveHello) [inline chkdm:89]: same here

**Review by PeterDaveHello — CHANGES_REQUESTED** [review]
Why's the order shuffled?

**Comment by thomasmerz** [comment]
⚠️  Do not merge, because problem to be fixed was a temporary one. ⚠️ 

Regarding your questions:
- I sorted the list alphabetically
- dnsforge.de came from https://github.com/PeterDaveHello/chkdomain/pull/14 which was closed

I'm fine with your questions 👍🏼

**Pull request closed without merging**